packages/validator/src/slashingProtection/minMaxSurround/minMaxSurround.ts

```diff
@@ -47,7 +47,7 @@ export class MinMaxSurround implements IMinMaxSurround {
   private async assertNotSurrounding(pubKey: BLSPubkey, attestation: MinMaxSurroundAttestation): Promise<void> {
     const minSpan = await this.store.minSpan.get(pubKey, attestation.sourceEpoch);
     const distance = attestation.targetEpoch - attestation.sourceEpoch;
-    if (minSpan != null && minSpan > 0 && minSpan < distance) {
+    if (minSpan !== null && minSpan > 0 && minSpan < distance) {
```

Comment:
type of miinSpan is `number | null`. It can't have the value `undefined`, making this diff safe